### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by jahagirdar** [aaiyer/bugseverywhere#2]
I had a few idea's for a BE and wanted to know whether it is still supported or not.

1. Integration with bugwarrior, and eventually use it to bi-directionally link the bugdatabase to UI's like trello etc.
2. Build an LSP or equivalent to enable using repo and history information to help auto complete bug information
e.g. 
1. use `git shortlog -esn` to extract contributor names for assigning bugs.
2. expand <leader>branch to insert branch/SHA id information
3. <leader>cmd to display a select list of the last-n commands from history
4. <leader>pwd for insert working directory
etc.

**Comment by aaiyer** [comment]
Yes it is. I maintain it sporadically, fix bug and review any PRs that come by.

Please go ahead and create a PR for any feature you might have, also open an issue when you start working on a feature - I will try to help also when I have time.

**Comment by tbm** [comment]
@aaiyer is this the official repo for Bugs Everywhere or did you fork it?

https://bugseverywhere.org/ needs updated. It points to a repo that no longer exists.

**Comment by tbm** [comment]
Hmm seems there's also https://gitlab.com/bugseverywhere/bugseverywhere

**Comment by mofosyne** [comment]
https://bugseverywhere.org/ appears to be squatted now. I wonder how it compares against https://github.com/git-bug/git-bug . One of my issue in deciding to try this or not is that I don't see any UI example to know what I am in for if I want to use it and how it compares against other approaches like git-bug or a standard github bug tracker.
